spec/numberformat.html

```diff
@@ -1251,5 +1506,313 @@ <h1>ComputeExponentForMagnitude ( _numberFormat_, _magnitude_ )</h1>
+          </thead>
+          <tr>
+            <td rowspan="2">*"ceil"*</td>
+            <td>positive</td>
+            <td>~infinity~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~zero~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"floor"*</td>
+            <td>positive</td>
+            <td>~zero~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~infinity~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"expand"*</td>
+            <td>positive</td>
+            <td>~infinity~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~infinity~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"trunc"*</td>
+            <td>positive</td>
+            <td>~zero~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~zero~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"halfCeil"*</td>
+            <td>positive</td>
+            <td>~half-infinity~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~half-zero~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"halfFloor"*</td>
+            <td>positive</td>
+            <td>~half-zero~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~half-infinity~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"halfExpand"*</td>
+            <td>positive</td>
+            <td>~half-infinity~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~half-infinity~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"halfTrunc"*</td>
+            <td>positive</td>
+            <td>~half-zero~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~half-zero~</td>
+          </tr>
+          <tr>
+            <td rowspan="2">*"halfEven"*</td>
+            <td>positive</td>
+            <td>~half-even~</td>
+          </tr>
+          <tr>
+            <td>negative</td>
+            <td>~half-even~</td>
+          </tr>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-applyunsignedroundingmode" aoid="ApplyUnsignedRoundingMode">
+      <h1>ApplyUnsignedRoundingMode ( _x_, _r1_, _r2_, _unsignedRoundingMode_ )</h1>
+      <p>
+        The abstract operation ApplyUnsignedRoundingMode considers _x_ (a mathematical value), bracketed below by _r1_ (a mathematical value) and above by _r2_ (a mathematical value), and returns either _r1_ or _r2_ according to _unsignedRoundingMode_ (a specification type from the *Unsigned Rounding Mode* column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> or *undefined*). The following steps are taken:
+      </p>
+      <emu-alg>
+        1. If _x_ is equal to _r1_, return _r1_.
+        1. Assert: _r1_ &lt; _x_ &lt; _r2_.
```

Comment:
TIL about `emu-eqn`! It's very nifty, could it be used for these comparisons as well?